setup.cfg

```diff
@@ -50,6 +50,8 @@ tensorflow =
 tensornets =
     tensorflow>=2.2.0
     tensornets
+reinforcement =
+    tf-agents[reverb]
```

Comment:
I couldn't get it to work by installing tf-agents[reverb]
This might be an issue local to my VM but I had to separately conda install dm-reverb, tf-agents and possibly tensorflow (I think I already had that installed)

Reply:
This might impact test/docs failures but I'm not sure...

Reply:
I think this is likely where the tests are failing. I think the tests would work if we forced numpy==1.23.5, but I didn't want to include that in the cf file. We could also try tf-agents-nightly, but this might reduce stability going forward.